src/main/java/org/apache/maven/plugins/pmd/AbstractPmdViolationCheckMojo.java

```diff
@@ -87,6 +87,14 @@
     @Parameter( property = "pmd.excludeFromFailureFile", defaultValue = "" )
     private String excludeFromFailureFile;
 
+    /**
+     * The maximum number of violations allowed before execution fails.
+     *
+     * @since 3.10
```

Comment:
Minor thing: With 3.9.0 we switched to a 3-part version number, so we should use here 3.10.0.